orangecontrib/spectroscopy/agilent.py

```diff
@@ -98,8 +98,9 @@ def _get_prop_d(dat, param):
         return val
 
     def _get_prop_str(dat, param):
```

Comment:
I would have named this internal helpers differently: I would not use the underscore. Also, "get" can often seems a bit superfluous.